### DISCUSSION TITLE
Recognize obsolete ignore list entries

### DISCUSSION THREAD

**Discussion by 3XC1T3D** [aquasecurity/trivy#4425] [Q&A] [answered]
Hi,

is there a possibility in trivy that you can recognize obsolete ignore list entries?

For example i ignore a several cve till january. After january trivy warns me that it recognised a cve. But if the cve was already fixed, trivy don't warn me anymore. But the old ignorelist entry is still there if i dont clean it up.

I hope i explanied it the correct way. If not, let me now 😅

**Comment by knqyf263** [comment, marked as answer]
Is this the same?
https://github.com/aquasecurity/trivy/discussions/4390

**Reply by 3XC1T3D** [reply]
> Is this the same?
> https://github.com/aquasecurity/trivy/discussions/4390

Yes, it is the same Question/Idea 😅

But if i am right, there is actual no solution 😅
